### DISCUSSION TITLE
The `platformio` command is not being recognized in Windows

### DISCUSSION THREAD

**Discussion by tqrahman** [Code-and-Response/ClusterDuck-Protocol#420] [QuackOverflow]
When running a `platformio` command such as `platformio --version`, the command prompt throws that issue.

**Comment by hawk4031** [comment]
It may need to be added on the PATH

**Comment by tqrahman** [comment]
The error us solved by adding the platformio scripts into the environment path in windows. 

On the Windows taskbar, right-click the Windows icon and select System. In the Settings window, under Related Settings, click Advanced system settings. On the Advanced tab, click Environment Variables. Click edit Path -> New. Then add `C:\Users\[YOUR USER]\.platformio\penv\Scripts\`. 

In our case, our user was `pixel` so the path would be `C:\Users\pixel\.platformio\penv\Scripts\`. 

![image](https://github.com/ClusterDuck-Protocol/ClusterDuck-Protocol/assets/17665316/ee69a49e-a290-4dba-900a-fa5e03719719)

For more information check out the PIO installation guide for Windows: https://docs.platformio.org/en/latest/core/installation/shell-commands.html#windows
